### DISCUSSION TITLE
thanhle.blog/en/blog/boost-nextjs-tti-fid-performance-without-compromise-and-pain

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#37] [Ideas]
# thanhle.blog/en/blog/boost-nextjs-tti-fid-performance-without-compromise-and-pain

https://thanhle.blog/en/blog/boost-nextjs-tti-fid-performance-without-compromise-and-pain

**Comment by darkpiv** [comment]
đã star và tín dụng
